ecl/agentexec/agentexec.cpp

```diff
@@ -386,6 +401,7 @@ class WaitThread : public CInterfaceOf<IPooledThread>
     StringAttr queue;
     Linked<IJobQueueItem> item;
     bool isThorAgent = false;
+    unsigned myInstanceNumber{};
```

Comment:
not sure what I think about that style :)
I think could be ok in some cases, but my feeling here is that a {0} is more explicit and readable.
But should also be same style where possible to other initializes in same class.
I'm okay with changing isThorAgent to {false} at same time.